### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by principlebrothers**
Design Awesome books shelve where users can;
- add books to the shelves (object array);
- remove books from shelves
- and the information added is stored in the local storage.

### CODE REVIEW

File: localStorage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    inputAuthor.value = formObject.author;
+  }
+}
+
+window.onload = () => {
+  getData();
+};
+
+inputTitle.onchange = storedData;
+inputAuthor.onchange = storedData;
```
Comment (keroloussamy) [inline localStorage.js]: - [ ] Kindly when I tried to add some books, the books didn't get saved on the local storage.

image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  const author = document.querySelector('#author').value;
+  addBook(title, author);
+  displayBooks();
+});
+
+function removeBook(selectedBook) {
+  const newBookList = books.filter((book) => book.title !== selectedBook);
+  books = newBookList;
+  return books;
+}
```
Comment (keroloussamy) [inline script.js]: - [ ] Kindly when I tried to remove one book, the whole list get deleted from the page kindly fix this issue.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: localStorage.js (hunk 1 above)
Comment (principlebrothers) [reply localStorage.js]: @keroloussamy We appreciate your feedback. We will like to confirm that the local storage is functional. 
Attached is an image of the local storage display.
evidence

File: script.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  });
+}
+
+addButton.addEventListener('click', () => {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  addBook(title, author);
+  displayBooks();
+});
+
```
Comment (brytebee) [inline script.js:54]: - [ ] **Required:** Great job on making some changes in regards to what the previous reviewer suggested. However, there is still more work to be done.

localstorage

1. The localstorage only save a single item per time.
2. A new item replaces the previous.
3. There is no attribute to iterate through your data.
4. On load & reload, the data is not rendered.

Fix these and request another review from your dashboard.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @principlebrothers & @keroloussamy👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: script.js (hunk 3 above)
Comment (principlebrothers) [reply script.js:54]: Hello @brytebee, thanks for the feedback. However, we as a team would appreciate a call or a video explanation of what you required that we implement. Thank you once again for your cooperation.

**Review by Davidosky007 — APPROVED** [review]
Hi @principlebrothers 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**